datajunction-server/datajunction_server/api/helpers.py

```diff
@@ -897,3 +899,101 @@ async def query_event_stream(  # pylint: disable=too-many-arguments
 
             query = query_next
         await asyncio.sleep(stream_delay)  # pragma: no cover
+
+
+def deactivate_node(session: Session, name: str, message: str = None):
```

Comment:
Should we try and stay consistent with delete/restore? Or did you mean to use delete/restore terminology on the client side only?

Reply:
Yeah, so I figured internally we can be a bit more clear that it's just deactivating and not deleting, but externally (i.e., anything that's potentially user-facing) we should call it "delete" and "restore".